frontend/packages/kubevirt-plugin/src/constants/vm/storage/data-volume-source-type.ts

```diff
@@ -11,20 +11,20 @@ export class DataVolumeSourceType extends SelectDropdownObjectEnum<string> {
   });
   static readonly HTTP = new DataVolumeSourceType('http', {
     // t('kubevirt-plugin~Import via URL (creates PVC)')
```

Comment:
in order for the translator to catch the changes in translation, you need to match the strings between the comments and the label key